tests/persistance/test_exceptions.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from strawberry.persistance.exceptions import PersistedQueryNotFoundError
+
+
+def test_notfound_exception_error_and_message():
+    with pytest.raises(Exception) as exc:
+        raise PersistedQueryNotFoundError()
+
+    assert exc.value.code == "PERSISTED_QUERY_NOT_FOUND"
+    assert exc.value.message == "PersistedQueryNotFound"
+
+
+def test_found_exception_instance():
```

Comment:
🤔don't think we need this test...  would be better to have it check in an 'integration' test if it's possible :)